### PR TITLE
Docs: Update install instructions

### PR DESCRIPTION
**Pull request by rschu1ze**
Motivated by discussion in community Slack:
https://clickhousedb.slack.com/archives/CU478UEQZ/p1714610641727259

### Changelog category (leave one):
- Documentation (changelog entry is not required)

### CODE REVIEW

**Comment by robot-ch-test-poll1** [comment]
*This is an automated comment for commit 3dc0441293a10f6ad3ca1773da0372ba6f3479d3 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/63267/3dc0441293a10f6ad3ca1773da0372ba6f3479d3/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>PR Check</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>✅ success</td></tr>
<tbody>
</table>
</details>

File: docs/en/getting-started/install.md [hunk 1]
```diff
@@ -38,31 +38,43 @@ On Linux, macOS and FreeBSD:
...
-1. Run the following command to start the ClickHouse server:
+2. Run the following command to start [clickhouse-local](../operations/utilities/clickhouse-local.md):
 
     ```bash
-    ./clickhouse server
+    ./clickhouse
     ```
 
-   The first time you run this script, the necessary files and folders are created in the current directory, then the server starts.
+   Clickhouse-local allows you to process local and remote files using ClickHouse's powerful SQL and without a need for configuration. Table
```
Comment (evillique) [inline docs/en/getting-started/install.md:47]: Maybe `Clickhouse-local` should start with the lowercase and be inside backticks to make it look ok?

Comment (rschu1ze) [reply docs/en/getting-started/install.md:47]: Good idea, done.

**Comment by rschu1ze** [comment]
(sorry, I accidentally amended my commit, this needs a force-push)

**Review by evillique — APPROVED** [review]

**Pull request merged**